lib/github_fetcher.rb

```diff
@@ -151,15 +151,34 @@ def ignored_ci_repos
     YAML.load_file(File.join(File.dirname(__FILE__), "../ignored_ci_repos.yml"))
   end
 
+  def is_ruby_app?(repo)
```

Comment:
Could we also follow the style guide https://rubystyle.guide/#bool-methods-prefix
```suggestion
  def rails_app?(repo)
```